lib/screens/models/patient_view.dart

```diff
@@ -13,7 +13,7 @@ class PatientModel {
   }
 
   String get uuid {
-    return _patient.id?.value ?? '';
+    return _patient.fhirId?.toString() ?? '';
```

Comment:
similar question ..
patient.id? is more understandable from _patient.fhirId

Reply:
The variable fhirId is declared everywhere of type string? 
Using id? gives error that id is not declared